apps/app/src/views/routes/Login.tsx

```diff
@@ -62,8 +62,8 @@ export const Login = () => {
               </Alert>
             ) : null}
             <HStack spacing="1" justify="center">
-              <Text color="muted">{`Don't have an account?`}</Text>
-              <Link color="teal.500" href="mailto:rado@photon.health">
+              <Text color="muted">Don't have an account?</Text>
+              <Link color="teal.500" href="mailto:sales@photon.health">
```

Comment:
I dont think this email is setup yet. You could do hello@photon.health

Reply:
Otto just set it up https://photonhealth.slack.com/archives/C02NJPUFVPV/p1707508842316309?thread_ts=1707496542.298209&cid=C02NJPUFVPV